draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -177,7 +177,7 @@ While the client may be able to deduce the operational state of the local server
 Within RADIUS, proxies typically only forward traffic between the NAS and RADIUS servers, and they do not generate their own response.
 As a result, when a NAS does not receive a response to a request, this could be the result of packet loss between the NAS and proxy, a problem on the proxy, loss between the RADIUS proxy and server, or a problem with the server.
 
-The absence of a reply can cause a client to deduce (incorrectly) that the proxy is unavailable.
+The absence of a reply can cause a client to deduce incorrectly that the proxy is unavailable.
```

Comment:
I would prefer leaving the parentheses in.
The deduction can be correct, if the server is actually down, but it may also be incorrect. The parentheses emphasize that the deduction is not always wrong.